### PR TITLE
Update Boilerplate

### PR DESCRIPTION
**Pull request by RobLoach**
:warning: Getting some inconsistancies with the dependencies paths.

Output: ["/home/rob/jstransformer-less/test/import/imported.less"]
   Expected:    ["test/import/imported.less"]
     ✗ less.render() (18ms)

```
   AssertionError: [ '/home/rob/jstransformer-less/test/import/imported.less' ] deepEqual [ 'test/import/imported.less' ]
```
- [ ] Squash the commits so that there's no history.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
 # jstransformer-less
 
-Transform less to css
-
-Options (in addition to those that are passed through to less):
-
- - if plugins is an array, it can contain string names like `'npm-import'` or `'inline-urls'` and this module will automatically `require('less-plugin-' + name)` for you.
```
Comment (ForbesLindesay) [inline README.md:7]: These extra options need to be documented somewhere.

**Comment by ForbesLindesay** [comment]
Can we rebase these "update to boilerplate" pull requests down to a short list of commits that actually show what's changed.  Looking at the changed files it looks like it could just be:
- Update .gitignore
- Add more platforms to .travis.yml
- Add change log in HISTORY.md
- Update LICENSE formatting
- Update description in package.json and README
- Update README with API example usage
- Update dependencies in package.json
- Use test-jstransformer for testing
- Add "files" array to package.json
- Add code coverage & badge

File: README.md (hunk 1 above)
Comment (RobLoach) [inline README.md:7]: Good find, restored: https://github.com/jstransformers/jstransformer-less/commit/4d613d3824a399944977274d2430d77a17319588

**Comment by RobLoach** [comment]
> Can we rebase these "update to boilerplate" pull requests down to a short list of commits that actually show what's changed.

Did a rebase, and was getting a lot of conflicts through the transition. Would there be an easier way to get this in?

**Comment by RobLoach** [comment]
Squash, or merge, the commits so that there's no history.

**Pull request closed without merging**